### PR TITLE
CORE-53717: Add new getConnectivity.php api

### PR DESCRIPTION
**Pull request by jgehrett**
- This will retrieve the configuration from settings/connectivity.ini
  in json, xml or html format
- Add this new link to admin_header.inc

### CODE REVIEW

**Comment by jgehrett** [comment]
getconnectivity_php_json
getconnectivity_php_xml
connectivity_config

**Comment by renajohn** [comment]
:+1:

**Comment by ocr** [comment]
@jgehrett, Renault gave you a +1, should we go ahead and merge it?

**Comment by jgehrett** [comment]
Not quite yet. I want to revisit the isDefault flag that was added. I think I want to move the logic into shepherd as that's the only place where its relevant.

**Comment by jgehrett** [comment]
@renajohn Ok, this is ready for another review. Thanks.

File: www/getConnectivity.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  echo "<tr>
+          <th class=\"location\">name</th>
+          <th>label</th>
+          <th>bwIn</th>
+          <th>bwOut</th>
+          <th>latency</th>
+          <th>plr</th>
+        </tr>\n";
+  foreach( $connectivities as $name => &$connectivity ) {
+    $error = '';
```
Comment (renajohn) [inline www/getConnectivity.php]: $error doesn't seems to be used.

**Comment by renajohn** [comment]
Except for the useless `$error`, you are good to go. :+1:

**Comment by jgehrett** [comment]
I've removed the unused `$error` from `getConnectivity.php`.

**Pull request merged**